src/promptflow/tests/conftest.py

```diff
@@ -20,7 +20,8 @@
 from dotenv import load_dotenv
 from filelock import FileLock
 from pytest_mock import MockerFixture
-from sdk_cli_azure_test.recording_utilities import SanitizedValues, is_replay
+from sdk_cli_azure_test.recording_utilities import SanitizedValues, is_record, is_replay
```

Comment:
in PR description pls include what's the main change to enable metrics support

Reply:
Okay, this is quite a long story...

Reply:
I'll make it after next iteration.